lib/browser/rpc-server.js

```diff
@@ -237,25 +237,24 @@ const unwrapArgs = function (sender, args) {
 // Call a function and send reply asynchronously if it's a an asynchronous
 // style function and the caller didn't pass a callback.
 const callFunction = function (event, func, caller, args) {
-  let err, funcMarkedAsync, funcName, funcPassedCallback, ref, ret
-  funcMarkedAsync = v8Util.getHiddenValue(func, 'asynchronous')
-  funcPassedCallback = typeof args[args.length - 1] === 'function'
+  const funcMarkedAsync = v8Util.getHiddenValue(func, 'asynchronous')
+  const funcPassedCallback = typeof args[args.length - 1] === 'function'
   try {
     if (funcMarkedAsync && !funcPassedCallback) {
       args.push(function (ret) {
         event.returnValue = valueToMeta(event.sender, ret, true)
       })
       func.apply(caller, args)
     } else {
-      ret = func.apply(caller, args)
+      const ret = func.apply(caller, args)
       event.returnValue = valueToMeta(event.sender, ret, true)
     }
   } catch (error) {
     // Catch functions thrown further down in function invocation and wrap
     // them with the function name so it's easier to trace things like
     // `Error processing argument -1.`
-    funcName = ((ref = func.name) != null) ? ref : 'anonymous'
-    err = new Error(`Could not call remote function '${funcName}'. Check that the function signature is correct. Underlying error: ${error.message}`)
+    const funcName = func.name || 'anonymous'
```

Comment:
This behaviour is a little different when `func.name` is the empty string. Can you confirm that the behaviour change is OK?

Reply:
The old variant would treat an empty string as a "good" value for `funcName`, which doesn't make much sense. So I 👍 the change.